lib/facter/mco_config.rb

```diff
@@ -2,7 +2,13 @@
   Facter.add("mco_#{node}_config") do
     setcode do
       config = nil
-      ["/etc/puppetlabs/mcollective/#{node}.cfg", "/etc/mcollective/#{node}.cfg"].each do |cfg|
+      if Facter.fact(:kernel).value =~ /windows/i
+        appData = Facter.fact(:common_appdata)
```

Comment:
@cyberious can you do a `puppet module list` - we are trying to determine where this fact exists. I don't see it either.